### DISCUSSION TITLE
Fana TV

### DISCUSSION THREAD

**Discussion by Eyob111** [iptv-org/community#832] [Q&A]
### Channel ID

FanaTV.et

### Notes

Fana tv is an Ethiopian channel and is free to the public even though they have a streaming website but I can't find stable IPTV from it. it lasts no longer than a day in m3u, So if you have a solution it would be great.

https://www.fanabc.com/english/live/

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by BellezaEmporium** [comment]
https://player.mayya.et/stream_url/fana

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19814
